### PR TITLE
Tpwdcee 5977

### PR DESCRIPTION
**Pull request by flavius**

### CODE REVIEW

File: etc/crontab.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Cron:etc/crontab.xsd">
     <group id="default">
         <job name="wirecard-update-payments" instance="Wirecard\ElasticEngine\Gateway\Model\TransactionUpdater" method="run">
-            <schedule>*/15 * * * *</schedule>
+            <schedule>*/2 * * * *</schedule>
```
Comment (flavius) [inline etc/crontab.xml]: This is set to 2 minutes to speed up testing, but it should be set back to 15 after testing.

**Comment by rinnhofer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/wirecard/magento2-ee/pullRequest?prid=4975402&bid=16431927)

**Comment by rinnhofer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/wirecard/magento2-ee/pullRequest?prid=4975402&bid=16431927)

**Comment by rinnhofer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/wirecard/magento2-ee/pullRequest?prid=4975402&bid=16431927)

**Pull request closed without merging**